generator/run_this.py

```diff
@@ -1,5 +1,3 @@
-from __future__ import print_function, unicode_literals
```

Comment:
Can we remove this file? I believe this is only used as the initial call to the generator so if we have the new build file, this shouldn't be needed anymore right?